front/app/components/admin/Table/Tr.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export interface Props {
+  children: React.ReactNode;
+  className?: string;
+}
+
+const Row = React.forwardRef(
+  (
+    { children, className }: Props,
+    ref: React.RefObject<HTMLTableRowElement>
```

Comment:
This is needed to support re-ordering, I guess?

Reply:
Yeah, although I think we could also just rename `ref` to `innerRef` or something and avoid this `forwardRef` stuff... what do you think?

Reply:
I think it's fine to leave it as is, tbh. No need to introduce custom props.